src/status/formatter.js

```diff
@@ -1,14 +1,32 @@
-module.exports = (info, userIdToUsername) => {
-  return [...info.values()]
+module.exports = ({ eventStatus, memberStatus }, userIdToUsername) => {
+  
+  return [...memberStatus.values()]
     .filter(member => userIdToUsername.has(member.id))
     .sort((a, b) => a.participated / a.notified >= b.participated / b.notified)
     .map(member => {
       const name = userIdToUsername.get(member.id);
       const missed = member.notified - member.participated;
+      const meetingsMissedCurrentMonth = member.notifiedCurrentMonth - member.participatedCurrentMonth
+      const meetingsMissedCurrentWeek = member.notifiedCurrentWeek - member.participatedCurrentWeek
       const participation_rate = Math.floor(
         member.participated * 100 / member.notified
       );
-      return `${name} has missed ${missed} meetings, ${participation_rate}% participation rate`;
+      return `${name} has missed:
+        This month: ${meetingsMissedCurrentMonth} meetings
+        This week:  ${meetingsMissedCurrentWeek} meetings
+        In total: ${missed} meetings
+        Participation rate: ${participation_rate}%`;
     })
-    .join('\n');
+    .concat(
```

Comment:
this can be its own function, the logic in the `formatter` is getting complicated and hard to reason about.